### ISSUE TITLE
getPeerSupportedProtocols: TypeError: Cannot read property 'message' of undefined

### ISSUE THREAD

**Issue by wemeetagain** [ChainSafe/lodestar#1403]
```
TypeError: Cannot read property 'message' of undefined
    at getPeerSupportedProtocols (/home/cayman/Code/lodestar/packages/lodestar/lib/sync/utils/sync.js:327:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Libp2pNetwork.<anonymous> (/home/cayman/Code/lodestar/packages/lodestar/lib/sync/reqResp/reqResp.js:
104:38)
```

**Comment by dapplion** [comment]
CC https://github.com/ChainSafe/lodestar/pull/1378

**Comment by wemeetagain** [comment]
Resolved by #1409
